app/services/bulk_upload/lettings/row_parser.rb

```diff
@@ -387,7 +408,11 @@ def validate_nulls
       next if log.optional_fields.include?(question.id)
       next if question.completed?(log)
 
-      fields.each { |field| errors.add(field, I18n.t("validations.not_answered", question: question.check_answer_label&.downcase)) }
+      fields.each do |field|
+        unless errors.any? { |e| fields.include?(e.attribute) }
```

Comment:
What is this doing?

Reply:
- yep so before if there was an error it was always get added to the relevant fields
- now if any one of the relevant fields has one error on it we no longer keeping adding errors to that grouping

eg `startdate` consists of `field_dd`, `field_mm`, `field_yy` from the mapping
if there is an error that already on `field_dd` the code will no longer surface any additional errors on those fields. this should prevent the user from being overwhelmed by too many errors